Source/effects.h

```diff
@@ -1065,6 +1065,8 @@ struct TSFX {
 extern int sfxdelay;
 extern _sfx_id sfxdnum;
 
+_sfx_id GetSoundByClass(int playerID, int16_t soundID);
+void PlaySoundByClass(int playerID, int16_t soundID, bool noRepeat = false);

```

Comment:
No reason for this to be exactly 16bit
```suggestion
_sfx_id GetSoundByClass(int playerID, int soundID);
void PlaySoundByClass(int playerID, int soundID, bool noRepeat = false);
```

Reply:
doesn't hurt and matches the sound enum type : P